### DISCUSSION TITLE
Is it possible to "unmerge" or repair a merged file that became corrupted?

### DISCUSSION THREAD

**Discussion by arnean** [mifi/lossless-cut#1558] [Support]
I merged some clips with different parameters. First one was 1920x1080 and others were 3840x2160. I deleted the source clips before checking the result. Now the resulting clip shows first part and then is blackness. Actually I do not need the first clip. Is it possible to "unmerge" the resulting clip and remove the first, unneeded part and then set the file format to 4K so that the resulting clip will be visible? Perhaps not with Lossless cut but with some other tools?

**Comment by mifi** [comment]
Not that I know, but you could try to cut away the first part (an maybe slightly into the second part), and see it can recover it.

**Comment by arnean** [comment]
Managed to fix it with the help of Lossless cut itself. I first tried to change the MP4 headers with hex editor and fix the dimensions of the clip - but it was not recognized at all. Apparently the problem was in the HEVC stream itself. Found a really nice tool to examine MP4 files https://gpac.github.io/mp4box.js/test/filereader.html Also Bento4 contained interesting utilities https://www.bento4.com/

Then I tried to follow your advice and cut away the bad part. But it didn't fix the stream - it was still identified as 1920x1080. Then an idea came that when the concatenation of the invalid clip screwed up the stream, perhaps the concatenation of the correctly sized clip will unscrew it. I took a small clip with my Gopro using correct settings and concatenated it in front of the damaged clip. And fixed it! Now I just removed the clip I just concatenate and got back my undamaged clip.

**Reply by mifi** [reply]
Nice! Thanks for reporting back
